Lib/idlelib/autocomplete.py

```diff
@@ -27,25 +26,20 @@
 
 class AutoComplete:
 
-    menudefs = [
-        ('edit', [
-            ("Show Completions", "<<force-open-completions>>"),
-        ])
-    ]
-
-    popupwait = idleConf.GetOption("extensions", "AutoComplete",
-                                   "popupwait", type="int", default=0)
-
-    def __init__(self, editwin=None):
+    def __init__(self, editwin):
         self.editwin = editwin
-        if editwin is not None:  # not in subprocess or test
-            self.text = editwin.text
-            self.autocompletewindow = None
-            # id of delayed call, and the index of the text insert when
-            # the delayed call was issued. If _delayed_completion_id is
-            # None, there is no delayed call.
-            self._delayed_completion_id = None
-            self._delayed_completion_index = None
+        self.reload()
+        self.text = editwin.text
+        self.autocompletewindow = None
+        # id of delayed call, and the index of the text insert when
+        # the delayed call was issued. If _delayed_completion_id is
+        # None, there is no delayed call.
+        self._delayed_completion_id = None
+        self._delayed_completion_index = None
+
+    def reload(self):
+        self.popupwait = idleConf.GetOption("main", "General",
+                                   "autocomplete_wait", type="int", default=0)
```

Comment:
I want these to be class methods called after the class is created.  I will fix for all 4 files. I also changed back to original args -- 'extensions', 'AutoComplete', 'popupwait'.